profiles/SampleBarbarian-setup.yaml

```diff
@@ -1,131 +1,604 @@
-# The other Supernatural skills are: Warding, Debilitation
-
-# Hunting settings
-training_manager_hunting_priority: true
+# NOTES:
+#    - You can find all the BASE settings inside the base.yaml within this same folder! Search the file for keywords for more settings!
+#    - If you can "TAP ADJECTIVE NOUN" for an item, then you can use that for the adjective & noun of the item within the scripts!
+
+# High-level example YAMLS:
+#  - Chuno
+
+# This file can be used with ;training-manager or ;t2.  The T2 specifics are at the very bottom.
+
+hometown: Crossing
+
+#all the settings to make you do allll the stuff in your safe room versus random rooms around town
+safe_room: 8246
+outdoor_room: 8246
+engineering_room: 8246
+outfitting_room: 8246
+enchanting_room: 8246
+
+#summon the janitor in combat or in safe-room when too cluttered
+dump_junk: true
+
+#bail on hunting if bleeding
+stop_hunting_if_bleeding: false
+
+#look for a minimum mana before settling on whats available
+hunting_room_min_mana: 10
+hunting_room_strict_mana: false
+
+#number of swings before switching weapons
+combat_trainer_action_count: 10
+#or number of mindstates - whichever happens first
+combat_trainer_target_increment: 3
+
+#default stance when entering combat-trainer
+#default_stance: # # #
+
+#where and how long to hunt
+hunting_info:
+  #can find zones with notes in base-hunting.yaml in the data file
+  # Note: You can add multiple "overflow" zones to the list.
+  # It checks each hunting zone for an empty room before waiting
+  # Example:
+  # :zone:
+  # - rats
+  # - louts
+  # - goblins
+- :zone:
+  - rats
+  - louts
+  #number of minutes to hunt if stop_on doesn't trigger first
+  :duration: 60
+  #will exit hunting if all of the skills lock - can list one or twenty-one skills
+  stop_on:
+  - Brawling
+  - Small Edged
+  #conditions for hunting
+  args:
+  #dance threshold, will dance if creatures in room are equal or less than the number
+  # - d#
+  #retreat threshold, will retreat if creatures in room are equal or more than the number
+  # - r#
+  #skills to end combat early if below threshold
+  stop_on_low:
+  #- Skill1
+
+# True means perform a hunt right away on startup. False means wait until a round of town-training is done.
+training_manager_hunting_priority: false
+# list of skills that will trigger a hunt loop if they go below the lower_limit. THIS MEANS ONLY COMBAT-SKILLS SHOULD BE ON HERE
 training_manager_priority_skills:
+- Small Edged
+- Light Thrown
+- Parry Ability
+
+#weapons to train first when you get to combat
+#use ranged if you use aiming_trainables to get the max number of skills moving quickly
+# Note: Turn off if ranged are outpacing the others!
+priority_weapons:
+  - Crossbow
+  - Bow
+  - Slings
+
+weapon_training:
+  #ranged/OffHand
+  Crossbow: forester's stonebow
+  Brawling: ''
+  Light Thrown: leather handbook
+  Heavy Thrown: diamondique hhr'ata
```

Comment:
❓ My thought was that since these are samples, they should default to store-bought items to make lich more accessible to newcomers (I'm assuming newcomers will start with the sample YAMLs). What do you think?

Reply:
that was just copied out of mine - it could be updated to normal stuff

Reply:
I agree, I was doing this with things like "shadows" to "backpack", etc. I'll make the changes.

Reply:
Actually, this kind of shows people that different names, punctuation, etc, can be used and how. Maybe leave it? I mean, a weapon name is a weapon name?